client/my-sites/stats/site.jsx

```diff
@@ -638,15 +628,15 @@ class StatsSite extends Component {
 							/>
 						) }
 					</div>
+					{ supportsPlanUsage && (
```

Comment:
It seems these lines were out of the ending </div> before. Are those changes intended?

Reply:
I think this was an unintended side effect of the ChartTabs refactoring, and they accidentally got moved inside that wrapper, which wasn't an intended part of the changes. Is that what you mean about being out of the ending before?